clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/AddExecutor.scala

```diff
@@ -36,37 +39,28 @@ class AddExecutor[TI <: TransferIndex](addCommand: AddCommand[TI])
       .flatMap(_.as[addCommand.index.MetadataType])
       .left
       .map { err =>
-        new RuntimeException(
-          s"Invalid metadata given at $location. Run the '${ClioCommand.getSchemaPrefix}$commandName' command to see the expected JSON format for ${name}s.",
+        new IllegalArgumentException(
+          s"Invalid metadata given at $location. Run the '${ClioCommand.getSchemaPrefix}$commandName' command to see the valid JSON for ${name}s.",
           err
         )
       }
 
     decodedOrError.fold(
-      Future
-        .failed(_) logErrorMsg s"Metadata at $location cannot be added to Clio", {
-        decoded =>
-          {
-            for {
-              existingMetadata <- if (!addCommand.force) {
-                queryForKey(webClient).logErrorMsg(
-                  s"Could not query the $name. No files have been added."
-                )
-              } else {
-                Future.successful(None)
-              }
-              upsertResponse <- addFiles(
-                webClient,
-                decoded,
-                existingMetadata
-              ).fold(
-                left => Future.failed(left),
-                identity
-              )
-            } yield {
-              upsertResponse
-            }
+      Future.failed, { decoded =>
+        for {
+          existingMetadata <- if (!addCommand.force) {
```

Comment:
Is there a way to make scalafmt put the `if` on its own line? It's confusing to me to have multi-line `if`s that don't start on a new line.

Reply:
No, but I can move the `if` into a `val` before the `for` begins